go/tunnels/id_generation.go

```diff
@@ -0,0 +1,24 @@
+package tunnels
+
+import (
+	"math/rand"
+	"strings"
+	"time"
+)
+
+var nouns = []string{"pond", "hill", "mountain", "field", "fog", "ant", "dog", "cat", "rabbit", "shoe", "campsite", "plane", "sofa", "chair", "library", "book", "ocean", "lake", "river", "horse"}
+var adjectives = []string{"fun", "happy", "interesting", "neat", "peaceful", "puzzeled", "thoughtful", "kind", "joyful", "overjoyed", "new", "giant", "sneaky", "quick", "majestic", "gleaming", "jolly", "fancy", "tidy", "marvelous", "glamorous", "swift", "silent", "amusing", "spiffy"}
```

Comment:
consider swapping out interesting, thoughtful, glamorous, overjoyed, gleaming, and marvelous.

Reply:
updated